runtime/testruntime/testruntime.go

```diff
@@ -224,11 +229,26 @@ func NewInstanceForProject(t TestingT, name string) (*runtime.Runtime, string) {
 	return rt, inst.ID
 }
 
-func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string) {
+func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string, error) {
 	rt := New(t)
 
 	_, currentFile, _, _ := goruntime.Caller(0)
 	projectPath := filepath.Join(currentFile, "..", "testdata", "ad_bids_druid")
+	var err error
+	creds := os.Getenv("METRICS_CREDS")
+	address := os.Getenv("METRICS_ADDRESS")
+	data, err := base64.StdEncoding.DecodeString(creds)
+	if err != nil {
+		return nil, "", err
+	}
```

Comment:
Can we replace this with a single env var `RILL_RUNTIME_DRUID_TEST_DSN`? And save it to our shared `.env` file so everyone on the team is able to run it (see ` rill devtool dotenv -h`)